ddgr

```diff
@@ -1709,6 +1842,7 @@ def parse_args(args=None, namespace=None):
     addarg('-r', '--reg', dest='region', default='us-en', metavar='REG',
            help="region-specific search e.g. 'us-en' for US (default); visit https://duckduckgo.com/params")
     addarg('-C', '--nocolor', dest='colorize', action='store_false', help='disable color output')
+    addarg('-I', '--noinstantanswer', dest='instant_answer', action='store_false', help='disable instant answer')
```

Comment:
As many things don't work, this should be disabled by default. In fact we can have `-i` to enable instant results.